NextCenturyCorporation/itm-evaluation-server
# ITM-195: Read in Metrics Eval scenario format

The following was accomplished for this PR:
- Updated API swagger to latest and re-generate model files
- Updated soartech and/or adept scenario for new format (adapted from [latest example](https://nextcentury.atlassian.net/wiki/download/attachments/3041951763/phase1-scenario.yaml?api=v2))
- Updated `ITMScenarioReader` to read new scenario file format and populate new model objects, including scenes
- Replaced probe reader with scene reader
- Deleted September-style scenarios and probes

To test:
- Connect a client to a running server and inspect the contents of the scenario and scenes after the call to `ITMSessionScenarioObjectHandler.read_scenario_from_yaml()`.  You can print them out, use a debugger, or whatever you wish. 
The server should reply to the client, but the client will crash because it hasn't been updated.
- Be sure to specify a `soartech` and `adept` client session (e.g., `--session soartech`), but not `eval`.  The adept scenario is the sample yaml file we've been maintaining, and the soartech scenario is a modified version of an pre-release scenario that soartech has provided to us.
